ToDoApp/Models/Info.swift

```diff
@@ -5,17 +5,19 @@
 //  Created by 최용헌 on 2023/04/12.
 //
 struct Info {
-    let title: String
-    let priority: String?
-    let memo: String
-    let date: String
-    var isCompleted: Bool
-    
-    init(title: String, priority: String?, memo: String, date: String = "", isCompleted: Bool) {
-        self.title = title
-        self.priority = priority
-        self.memo = memo
-        self.date = date
-        self.isCompleted = isCompleted
-    }
+  var id: Int
+  var title: String
+  var priority: String?
+  var memo: String
+  var date: String
+  var isCompleted: Bool
+  
+  init(id: Int, title: String, priority: String?, memo: String, date: String = "", isCompleted: Bool) {
+    self.id = id
+    self.title = title
+    self.priority = priority
+    self.memo = memo
+    self.date = date
+    self.isCompleted = isCompleted
```

Comment:
각 할 일들의 식별을 나타내주는 `Int`타입인 `id` 프로퍼티를 직접 주입하지 않고
각 인스턴스 고유의 식별을 나타내주는 `UUID`를 사용해주면 좋을 것 같아
이러면 개발자가 굳이 신경써서 각 할 일들의 식별을 신경써줄 필요가 없어 ! 

```suggestion
  var id: UUID
  var title: String
  var priority: String?
  var memo: String
  var date: String
  var isCompleted: Bool
  
  init(title: String, priority: String?, memo: String, date: String = "", isCompleted: Bool) {
    self.id = UUID()
    self.title = title
    self.priority = priority
    self.memo = memo
    self.date = date
    self.isCompleted = isCompleted
```